sonar-plugin-api/src/main/java/org/sonar/api/measures/CoreMetrics.java

```diff
@@ -1241,12 +1289,16 @@
 
   /**
    * @since 3.3
+   * @deprecated since 5.2 soon to be removed
    */
+  @Deprecated
   public static final String OVERALL_COVERED_CONDITIONS_BY_LINE_KEY = "overall_covered_conditions_by_line";
```

Comment:
:new: ![INFO](https://raw.githubusercontent.com/henryju/image-hosting/master/info.png)Do not forget to remove this deprecated code someday.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)